src/client/render/secondstage.h

```diff
@@ -22,17 +22,31 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "stereo.h"
 #include "pipeline.h"
 
+/// Step to apply post-processing filter to the rendered image
```

Comment:
Is it okay to use triple slash (`/`)?

Reply:
Yes, but I still aligned it with the rest of doxygen comments in the file.